packages/Modal/src/components/FocusTrap.js

```diff
@@ -0,0 +1,13 @@
+import Overlay from "@paprika/overlay";
+
+const propTypes = Overlay.propTypes.focusTrapOptions;
+
+const defaultProps = Overlay.defaultProps.focusTrapOptions;
```

Comment:
I know that this is consistent with our standard, but for such a small component, it seems like unnecessary indirection.  WDYT?